cmd/oras/push.go

```diff
@@ -127,30 +134,41 @@ func runPush(opts pushOptions) error {
 	return opts.ExportManifest(ctx, store, desc)
 }
 
-func packManifest(ctx context.Context, store *file.Store, annotations map[string]map[string]string, opts *pushOptions) (ocispec.Descriptor, error) {
+func packManifest(ctx context.Context, store *file.Store, annotations map[string]map[string]string, opts *pushOptions) (manifestDesc ocispec.Descriptor, err error) {
 	var packOpts oras.PackOptions
 	packOpts.ConfigAnnotations = annotations[annotationConfig]
 	packOpts.ManifestAnnotations = annotations[annotationManifest]
 
-	if opts.manifestConfigRef != "" {
-		path, mediatype := parseFileReference(opts.manifestConfigRef, oras.MediaTypeUnknownConfig)
-		desc, err := store.Add(ctx, annotationConfig, mediatype, path)
+	if len(opts.FileRefs) != 0 {
```

Comment:
Need more comment like
```suggestion
	if len(opts.FileRefs) != 0 {
	// pack OCI artifact
```